.ado/jobs/e2e-test.yml

```diff
@@ -44,6 +44,24 @@ jobs:
 
       - template: ../templates/prepare-env.yml
 
+      - task: CmdLine@2
```

Comment:
When this passes and is in to unblock us, I hope we can try to run the build without these changes to see if the nuget.config is enough to get past.

Reply:
I saw that you tried with only the nuget.config files and ist still failed, another idea might be to place a Nuget.Config file in the root of the repo.